packages/ui/src/theme/css/component/liveness.scss

```diff
@@ -97,12 +98,16 @@
 .amplify-liveness-recording-icon {
   flex-direction: column;
   align-items: center;
-  background-color: var(--amplify-colors-background-primary);
+  background-color: var(--amplify-colors-white);
```

Comment:
Is this change intentional? background-primary references white so there isn't really an effect by default, but would make the theme API harder to grok because the color white has no semantic meaning

Reply:
This was intentional to keep it white because Rekognition asked that the color not change with the color theme. They wanted it always to stay white